indico/modules/events/registration/client/js/form/fields/AccommodationInput.jsx

```diff
@@ -148,9 +138,55 @@ export default function AccommodationInput({htmlName, disabled, choices, arrival
   );
 }
 
+AccommodationInputComponent.propTypes = {
+  value: PropTypes.object.isRequired,
+  onChange: PropTypes.func.isRequired,
+  disabled: PropTypes.bool.isRequired,
+  choices: PropTypes.arrayOf(PropTypes.shape(choiceShape)).isRequired,
+  arrival: PropTypes.shape({
+    startDate: PropTypes.string.isRequired,
+    endDate: PropTypes.string.isRequired,
+  }).isRequired,
+  departure: PropTypes.shape({
+    startDate: PropTypes.string.isRequired,
+    endDate: PropTypes.string.isRequired,
+  }).isRequired,
+  // TODO: placesUsed, captions - only needed once we deal with real data
+};
+
+export default function AccommodationInput({
+  htmlName,
+  disabled,
+  isRequired,
+  defaultValue,
+  choices,
+  arrival,
+  departure,
+}) {
+  return (
+    <FinalField
+      name={htmlName}
+      component={AccommodationInputComponent}
+      required={isRequired}
+      disabled={disabled}
+      defaultValue={defaultValue}
+      choices={choices}
+      arrival={arrival}
+      departure={departure}
+      validate={value => {
+        if (!value.isNoAccommodation && (!value.arrivalDate || !value.departureDate)) {
+          return Translate.string('You must select the arrival and departure date');
+        }
+      }}
+    />
+  );
+}
+
 AccommodationInput.propTypes = {
   htmlName: PropTypes.string.isRequired,
   disabled: PropTypes.bool,
+  isRequired: PropTypes.bool,
+  defaultValue: PropTypes.object,
```

Comment:
I would not accept a default prop if we don't support it - otherwise it's just confusing when someone wonders where the prop could be coming from. The defaultValue from the default props below could go directly into the FinalField `defaultValue` above.